samples/blog/BlogArticles.scala

```diff
@@ -13,6 +13,11 @@ object BlogArticlesApi {
     case object DRAFT extends BlogArticlesApi.ArticleStatus
     case object PUBLISHED extends BlogArticlesApi.ArticleStatus
   }
-  case class Pagination(first: Int, count: Int, reverse: Option[Boolean])
+  case class Pagination(first: Int, count: Int, order: Option[PaginationOrder])
```

Comment:
They are generated for each test, so one `.graphql` file has a `.scala` file with the expected output of the generator.